libcudacxx/include/cuda/std/detail/libcxx/include/__string

```diff
@@ -264,9 +265,9 @@ struct _CCCL_TYPE_VISIBILITY_DEFAULT char_traits<char>
   _LIBCUDACXX_HIDE_FROM_ABI static size_t _CCCL_CONSTEXPR_CXX14 length(const char_type* __s) noexcept
   {
 #ifdef _CCCL_HAS_NO_BUILTIN_STRLEN
-#  ifdef _LIBCUDACXX_IS_CONSTANT_EVALUATED // is_constant_evaluated only exists since GCC 9
+#  ifdef _CCCL_BUILTIN_IS_CONSTANT_EVALUATED // is_constant_evaluated only exists since GCC 9
     if (__libcpp_is_constant_evaluated())
-#  endif // defined(_LIBCUDACXX_IS_CONSTANT_EVALUATED)
+#  endif // defined(_CCCL_BUILTIN_IS_CONSTANT_EVALUATED)
```

Comment:
why are we using `__libcpp_is_constant_evaluated()` instead of using `_CCCL_BUILTIN_IS_CONSTANT_EVALUATED()`?

Reply:
Actually in that case we could use it directly, most often `__libcpp_is_constant_evaluated` is used without conditionals